client/src/views/browse.html

```diff
@@ -0,0 +1,14 @@
+<div class="row">
+    <div class="small-12 text-center">
+        <br>
+        <b>Browse For Siafiles.</b>
+    </div>
+</div>
+
+<div data-filter></div>
+
+<div class="row">
+    <div class="large-8 medium-10 small-12 large-offset-2 medium-offset-1">
+        <table data-siafile-table></table>
+    </div>
+</div>
```

Comment:
Trying to make it super modularized. the directives are all reusable custom html elements.